test/controllers/meetups_controller_test.rb

```diff
@@ -1,7 +1,34 @@
 require 'test_helper'
 
 class MeetupsControllerTest < ActionDispatch::IntegrationTest
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'should show main page' do
+    get '/meetups/index'
+    assert_response :redirect
+    follow_redirect!
+    assert_response :success
+  end
+
+  test 'should redirect to meetup page if there is not id in params' do
+    get new_meetup_path
```

Comment:
Actually, you don't have to provide an id to go to `new_meetup_path`. The only requirement is that there's a logged in user, this means that `current_user` exists (we use [devise](https://github.com/plataformatec/devise)).

If the user has not logged in, he/she will be redirected to the `meetups_path`.

This validation is made through the [meetups policy](https://github.com/infusionvlc/infusion/blob/master/app/policies/meetup_policy.rb) thanks to the [pundit gem](https://github.com/varvet/pundit).